swizzle/inventory.ini.example

```diff
@@ -0,0 +1,23 @@
+machine-0 ansible_host=[server IP] ansible_user=[user] ansible_ssh_private_key_file=[path to ssh key]
```

Comment:
Should we include `ansible_ssh_common_args='-o StrictHostKeyChecking=no'` on each host? Had to set it to avoid the prompt.

Reply:
It's usually easier to include an `ansible.cfg` file in the cwd where we expect users to run ansible to set common config like this.

Reply:
I would personally prefer not to make this the default behavior.

Reply:
Agreed. Thanks!